sql/driver/conn.go

```diff
@@ -112,6 +109,9 @@ func (c *conn) send(args Request) (*rows, error) {
 		return r, nil
 	}
 	result := resp.Results[index]
+	if result.Error != nil {
```

Comment:
Silently dropping errors is extremely surprising. The justification for doing so needs to be high and included in a comment in the code. Note that combining the individual error messages into a single message would take something like 5 lines of code. I'm not sure I would classify that as fancy. Regardless, I think we might want to change the semantics for multi-statement requests so that they are all or nothing, just as single-statement requests are all or nothing.

My understanding of the postgres protocol is that the protocol itself does not support multi-statement requests. That is, each statement is a separate request.